modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-algorithms-benchmarking_{context}"]
+= Benchmarking Kopia hashing, encryption, and splitter algorithms
+
+You can run Kopia commands to benchmark the hashing, encryption, and splitter algorithms. Based on the benchmarking results, it is possible to select the most suitable algorithm for your workload. In this procedure, you run the Kopia benchmarking commands from a pod on the cluster. The benchmarking results can vary depending on CPU speed, available RAM, disk speed, current I/O load, and so on.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have an application with persistent volumes running in a separate namespace.
+* You have run a backup of the application with CSI snapshots.
+
+.Procedure
+
+. Configure a pod as shown in the following example. Make sure you are using the `oadp-mustgather` image for {oadp-short} version 1.3 and later.
++
+.Example pod configuration
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: oadp-mustgather-pod
+  labels:
+    purpose: user-interaction
+spec:
+  containers:
+  - name: oadp-mustgather-container
+    image: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.3
+    command: ["sleep"]
+    args: ["infinity"]
+----
++
+[NOTE]
+=====
+The Kopia client is available in the `oadp-mustgather` image.
+=====
++
+. Create the pod by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <pod_config_file_name> # <1>
+----
+<1> Specify the name of the YAML file for the pod configuration.
++
+. Verify that the Security Context Constraints (SCC) on the pod is `anyuid`, so that Kopia can connect to the repository.
++ 
+[source,terminal]
+----
+$ oc describe pod/oadp-mustgather-pod | grep scc
+----
++
+.Example output
++
+[source,terminal]
+----
+openshift.io/scc: anyuid
+----
++
+. Connect to the pod via SSH by running the following command:
++ 
+[source,terminal]
+----
+$ oc -n openshift-adp rsh pod/oadp-mustgather-pod
+----
++
+. Connect to the Kopia repository by running the following command:
++
+[source,terminal]
+----
+sh-5.1# kopia repository connect s3 \
+  --bucket=<bucket_name> \ # <1>
+  --prefix=velero/kopia/<application_namespace> \ # <2>
+  --password=static-passw0rd \ # <3>
+  --access-key="<access_key>" \ # <4>
+  --secret-access-key="<secret_access_key>" \ # <5>
+  --endpoint=<bucket_endpoint> \ # <6>
+----
+<1> Specify the object storage provider bucket name.
+<2> Specify the namespace for the application.
+<3> This is the Kopia password to connect to the repository.
+<4> Specify the object storage provider access key.
+<5> Specify the object storage provider secret access key.
+<6> Specify the bucket endpoint. You do not need to specify the bucket endpoint, if you are using AWS S3 as the storage provider.
```

Comment:
An attribute exists for AWS in it's short form. OADP docs also reference the `common-attributes.adoc` file?

![Screenshot from 2024-09-02 16-30-01](https://github.com/user-attachments/assets/003abf40-b8ee-4206-91a1-2922b0898b0b)

Reply:
done.